### ISSUE TITLE
Incorrect multiline import indent

### ISSUE THREAD

**Issue by Djaler** [vuejs/eslint-plugin-vue#1524]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:** 7.23.0
- **eslint-plugin-vue version:** 7.12.0 

**Please show your full configuration:**

```json5
module.exports = {
  extends: [
    'plugin:vue/recommended',
  ],
  plugins: ['vue'],
  rules: {
    'vue/script-indent': ['warn', 4, {
        baseIndent: 1,
        switchCase: 1,
    }],
  },
};
```

**What did you do?**	

```vue
<script lang="ts">
    import {
        computed, defineComponent, PropType, ref, watch,
    } from '@vue/composition-api';
```

**What did you expect to happen?**
Nothing

**What actually happened?**
`15:1  warning  Expected indentation of 8 spaces but found 4 spaces  vue/script-indent`

Temporary workaround - add 'ImportDeclaration' to `ignores` option of `vue/script-indent` rule.

**Comment by Djaler** [comment]
Looks like this was broken in https://github.com/vuejs/eslint-plugin-vue/pull/1520, because 7.11.1 works good in this case

**Comment by Djaler** [comment]
Wow. That was fast! Thanks

**Comment by ota-meshi** [comment]
Thanks for reporting this bug! I fixed and released it.
https://github.com/vuejs/eslint-plugin-vue/releases/tag/v7.12.1
